### ISSUE TITLE
Ice Box not converting Gas To Liquid on Create 0.5.1.h

### ISSUE THREAD

**Issue by KalWantsPizza** [lightning2401/northstar#80]
There's an issue either with the Ice Box or the Temp Regulator in the latest version of Create (0.5.1.h). Neither Hydrogen nor Oxygen gas will convert to liquid when at the right mode and temperature.

However, rolling the Create version back to 0.5.1.f will fix this, and it converts immediately with no other changes.

**Comment by LucaBiada** [comment]
yes, encountered the same bug, i wonder if there's a fix in the making
